integration/src/main/resources/suites/local/FullIntegration.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "FullIntegration",
+  "description": "All integration tests",
+  "serverSpecificationFile": "catalog-local.json",
```

Comment:
Where is `catalog-local.json` stored?

Reply:
It already existed; it's the same as `dev` but using `localhost` for the catalog service.